src/index.js

```diff
@@ -269,7 +272,7 @@ client.on('messageCreate', message => {
               channelMembersWithAccess = await client.guilds.cache.get(server.id).channels.fetch(alert.channelId).then(channel => (channel.members.filter((member) => ((member.permissions.serialize().KICK_MEMBERS || hardCodedApplePerms(member)) && member.presence !== null && member.presence?.status !== 'offline' && member.user?.bot === false)))),
               channelMembersWithAccessAll = await client.guilds.cache.get(server.id).channels.fetch(alert.channelId).then(channel => (channel.members.filter((member) => ((member.permissions.serialize().KICK_MEMBERS || hardCodedApplePerms(member)) && member.user?.bot === false)))),
               serverThreads = await client.guilds.cache.get(server.id).channels.fetch(alert.channelId).then((threads) => threads.threads),
-              matchingThread = await serverThreads.fetchArchived().then(thread => thread.threads.filter(y => y.name.split(/ +/g).slice(-1)[0] === member.id).first());
+              matchingThread = await serverThreads.fetch().then(thread => thread.threads.filter(y => y.name.split(/ +/g).slice(-1)[0] === member.id).first());
```

Comment:
This will cause issues as it won't find old archived threads.

Reply:
Otherwise it will cause issues as it won't find open threads

Reply:
I guess about merging them?

Reply:
might be better to use functions from userCases.js instead, which also use existing archived threads (because fetch() either return active or archived threads)

Reply:
I agree, either someone else can do that or I can do that when I get home